phpstan.neon

```diff
@@ -3,3 +3,5 @@ parameters:
         - PhpcrMigration
         - Tests
     level: max
+    ignoreErrors:
+        - '#caught "Throwable" must be rethrown#'
```

Comment:
Add phpstan ignore comment on the specific line, as soon as we have some ignored errors the list will only grow